test/api/v3/integration/payments/apple/GET-payments_apple_cancelSubscribe.js

```diff
@@ -0,0 +1,40 @@
+import {generateUser} from '../../../../../helpers/api-integration/v3';
+import applePayments from '../../../../../../website/server/libs/applePayments';
+
+describe('payments : apple #cancelSubscribe', () => {
+  let endpoint = '/iap/ios/subscribe/cancel';
```

Comment:
the enpoint should include the noRedirect query parameter because the home takes too long to load an may cause a test timeout